pkg/daemon/ceph/client/command.go

```diff
@@ -266,10 +266,9 @@ func ExecuteCephCommandWithRetry(
 				if exitCode == retryOnExitCode {
 					logger.Infof("command failed for %s. trying again...", action)
 					time.Sleep(waitTime)
-					continue
 				}
 			}
-			return nil, errors.Wrapf(err, "failed to complete command for %s", action)
+			continue
```

Comment:
If we're going to continue on any error, how about the following?
- Delete lines 264-270. No need to parse the exit code if we're always going to retry.
- Keep lines 267-268 before we continue on this line, so we can log the retry and then sleep.

Reply:
But then we have to wait for every error?

Reply:
Updated